driver/node.go

```diff
@@ -76,6 +82,34 @@ func (s *nodeService) NodePublishVolume(ctx context.Context, req *csi.NodePublis
 	s.mu.Lock()
 	defer s.mu.Unlock()
 
+	if volumeContext[ephVolConKey] == "true" {
+		listResp, err := s.client.GetLVList(ctx, &proto.Empty{})
+		if err != nil {
+			return nil, status.Errorf(codes.Internal, "failed to list LV: %v", err)
+		}
```

Comment:
This duplicates the code after the line 113.  Could you refactor a bit?

Reply:
done